src/lib/api.ts

```diff
@@ -40,11 +41,15 @@ export const getBoatAssignments = async () => {
 	return data;
 };
 
-export const getUserPastReservations = async (uid: string, maxDateStr: string) => {
+export const getUserPastReservations = async (maxDateStr: string) => {
+	const token = await auth.currentUser?.getIdToken();
 	const response = await fetch('/api/getUserPastReservations', {
 		method: 'POST',
-		headers: { 'Content-type': 'application/json' },
-		body: JSON.stringify({ user: uid, maxDateStr })
+		headers: {
+			'Content-type': 'application/json',
+			Authorization: 'Bearer ' + token
```

Comment:
@nrxszvo ideally all the ajax http requests will include this.

Reply:
got it